build.sh

```diff
@@ -84,13 +84,12 @@ if [ ! -z "$GIT_EMAIL" ]; then
     git config user.email "$GIT_EMAIL"
 fi
 if [ ! -z "$COMMIT_MESSAGE" ]; then
-    sed "s/\(^\s*s.version\s*= \).*$/\1$(cat version)/g" -i pointzi.podspec
+    sed -i '' 's/\(s.version[[:space:]]*=[[:space:]]"\).*/\1$(cat version)"/g' pointzi.podspec
```

Comment:
no need for the '' after the -i

Reply:
Checked from https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux, the third param '' is used for backup file, '' means no backup. When using sed mac os force we to specify a backup method. or we can use .bak to create a backup file for the original one.